src/firewall/core/io/firewalld_conf.py

```diff
@@ -331,3 +341,42 @@ def write(self):
             raise IOError("Failed to create '%s': %s" % (self.filename, msg))
         else:
             os.chmod(self.filename, 0o600)
+
+    @staticmethod
+    def _parse_reload_policy(value):
+        valid = True
+        result = {
+            "INPUT": "DROP",
+            "FORWARD": "DROP",
+            "OUTPUT": "DROP",
+        }
+        if value:
+            value = value.strip()
+            v = value.upper()
+            if v in ("ACCEPT", "REJECT", "DROP"):
+                for k in result:
+                    result[k] = v
+            else:
+                for a in value.replace(";", ",").split(","):
+                    a = a.strip()
+                    if not a:
+                        continue
+                    a2 = a.replace("=", ":").split(":", 2)
+                    if len(a2) != 2:
+                        valid = False
+                        continue
+                    k = a2[0].strip().upper()
+                    if k not in result:
+                        valid = False
+                        continue
+                    v = a2[1].strip().upper()
+                    if v not in ("ACCEPT", "REJECT", "DROP"):
+                        valid = False
+                        continue
+                    result[k] = v
+
+        return result, valid
```

Comment:
Please don't return multiple values. If it's invalid raise `FirewallError`.

Reply:
changed.